src/server/qgsserversettings.cpp

```diff
@@ -516,3 +570,31 @@ bool QgsServerSettings::logProfile()
 {
   return value( QgsServerSettingsEnv::QGIS_SERVER_LOG_PROFILE, false ).toBool();
 }
+
+QString QgsServerSettings::serviceUrl( const QString &service ) const
+{
+  QString result;
+  if ( service == "WMS" )
```

Comment:
We have to compare the string with the `compare(...)` method.

Note for myself: an enum for defining services would be useful

Reply:
Done using compare